scripts/build.js

```diff
@@ -11,12 +11,12 @@ const mapObj = require("map-obj");
 
 const overrides = require("./overrides");
 
-if (!process.env.ANICCA_REPOSITORY_PATH && !process.env.GITHUB_WORKSPACE) {
-  throw new Error("Please set ANICCA_REPOSITORY_PATH or GITHUB_WORKSPACE");
-}
-
 run();
 
+if (!process.env.GITHUB_ACTIONS && !process.env.ANICCA_REPOSITORY_PATH) {
+  throw new Error("Please set ANICCA_REPOSITORY_PATH");
+}
+
```

Comment:
can you revert this change? We have to check for the env variables before calling `run()`

Reply:
Moved lines before `run()`

—-

Be aware I modified the check considering it’s not necessary to check the existance of GITHUB_WORKSPACE env var. When my new code resolves aniccaCwd will check if it’s GitHub Action or not and if it is, that env var will exist by definition. This way we don’t enforce users to have that env var defined in local.